docs/resources/service-providers.md

```diff
@@ -56,25 +168,26 @@ gaiad config node <host>:<port
 // example: gaiad config node https://77.87.106.33:26657
 ```
 
-If you run your own full-node, just use `tcp://localhost:26657` as the address. 
+If you run your own Full Node locally, use `tcp://localhost:26657` as the address. 
```

Comment:
```suggestion
If you run your own full node locally, use `tcp://localhost:26657` as the address. 
```

Reply:
seems like we have more lower case instances of "full node" but let's pick one style and stick with it